webdriver/tests/element_click/link.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click Link Element Tests Step 7
+
+@pytest.mark.parametrize("element", ["<a href=\"/webdriver/tests/element_click/support/input.html\">123456</a>",
```

Comment:
You will want to test navigation in a different test.

Reply:
This is just a test to click a link with numbers and a link that is multi-line.

Reply:
Right, but it will still cause navigation?

Reply:
fixed